test/markup/diff/meta-string.txt

```diff
@@ -0,0 +1 @@
+index revA..revB file_mode
```

Comment:
This should preferably be a real example exactly as it would appear in an actual generated diff.